l10n_it_fatturapa/models/partner.py

```diff
@@ -70,30 +70,40 @@ def _check_ftpa_partner_data(self):
                         "As a natural person, partner %s "
                         "must have Name and Surname"
                     ) % partner.name)
-                if not partner.is_pa and (
-                    not partner.codice_destinatario or
-                        len(partner.codice_destinatario) != 7):
-                    raise ValidationError(_("Partner %s Addressee Code"
-                                            " must be 7 characters long")
-                                          % partner.name)
-                if not partner.vat and not partner.fiscalcode:
+                if not partner.country_id:
```

Comment:
Potresti uniformare i cambiamenti apportati in questo commit con quelli realizzati per la 10.0?

Reply:
fatto (spero)